### PR TITLE
gnome-calculator: add pt_BR translation

### PR DESCRIPTION
**Pull request by T16K**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/linux/gnome-calculator.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Iniciar a GNOME Calculator GUI:
+
+`gnome-calculator`
+
+- Resolver uma equação específica na linha de comando sem iniciar o aplicativo de desktop:
+
+`gnome-calculator --solve {{2^5 * 2 + 5}}`
+
+- Exibir a versão:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/gnome-calculator.md:14]:
```suggestion
- Exibe a versão:
```

File: pages.pt_BR/linux/gnome-calculator.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# gnome-calculator
+
+> A calculadora oficial para o ambiente de desktop GNOME.
+> Mais informações: <https://wiki.gnome.org/Apps/Calculator>.
+
+- Iniciar a GNOME Calculator GUI:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/gnome-calculator.md:6]: For Romance languages, the present indicative fits better.
```suggestion
- Inicia a GNOME Calculator GUI:
```

File: pages.pt_BR/linux/gnome-calculator.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`gnome-calculator`
+
+- Resolver uma equação específica na linha de comando sem iniciar o aplicativo de desktop:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/gnome-calculator.md:10]:
```suggestion
- Resolve uma equação específica na linha de comando sem iniciar o aplicativo de desktop:
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**